.circleci/config.yml

```diff
@@ -6,18 +6,8 @@ orbs:
 
 executors:
   validator:
-    docker:
-      - image: circleci/openjdk:11-jdk
-      - image: localstack/localstack:0.10.5
-        environment:
-          - SERVICES=sqs,sns
-          - DEBUG=${DEBUG- }
-          - DATA_DIR=/tmp/localstack/data
-          - DOCKER_HOST=unix:///var/run/docker.sock
-          - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR- }
-          - AWS_EXECUTION_ENV=True
-          - DEFAULT_REGION=eu-west-2
-          - TMPDIR=/private
+    machine:
+      image: ubuntu-2004:202101-01
     environment:
```

Comment:
we seem to need to use machine executor for testcontainers to work - https://www.testcontainers.org/supported_docker_environment/continuous_integration/circle_ci/